pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1344,8 +1344,8 @@ func (og *operationGenerator) GenerateVerifyControllerAttachedVolumeFunc(
 			}
 		}
 
-		// Volume not attached, return error. Caller will log and retry.
-		return volumeToMount.GenerateError("Volume not attached according to node status", nil)
+		// Volume not attached, do not return err. Kubelet will reconcile the actual and desired status.
+		return nil, nil
```

Comment:
@jingxu97 this is the return that would cause API usage.

Reply:
If return not nil, exponential backoff may cause the volume can not be mounted in time. The performance of attach fucnction is not sure. If the performance is very poor, it may cost very long. However, I think when Kebelet calls the mount function should not be affected by it.

Reply:
oh, sorry I missed this change. Can you just make the first change (return nil while checking volumeInUse) and see how much performance is improved? 

If we need further performance enhancement, there are another option we can consider: Decrease the maximum value of exponential backoff. Currently I think we set it to 2 mins, I think reduce to 15-30 seconds seems fine.

One more parameter I have been thinking to adjust is the reconciler looptime. 100ms seems too short, we end up having a lot go routine and just found there are existing operations pending.

Reply:
@jingxu97 Thank you for your advice. I have removed the 2rd place. I would observe the performance enhancement later.

> If we need further performance enhancement, there are another option we can consider: Decrease the maximum value of exponential backoff. Currently I think we set it to 2 mins, I think reduce to 15-30 seconds seems fine.
> 
> One more parameter I have been thinking to adjust is the reconciler looptime. 100ms seems too short, we end up having a lot go routine and just found there are existing operations pending.

I think it would be wonderful if exponential backoff for `verify` function can be set differently